src/SCRIPTS/BF/pages.lua

```diff
@@ -60,4 +60,8 @@ if apiVersion >= 1.16 then
     PageFiles[#PageFiles + 1] = { title = "Trim Accelerometer", script = "acc_trim.lua" }
 end
 
+if apiVersion >= 1.45 then
+    PageFiles[#PageFiles + 1] = { title = "OSD Pos and Profil", script = "pos_osd.lua" }
```

Comment:
Can we make it `OSD Pos and Profile`?  Otherwise maybe `OSD positioning`

```suggestion
    PageFiles[#PageFiles + 1] = { title = "OSD Pos and Profile", script = "pos_osd.lua" }
```

Reply:
Please use english in the title.

Reply:
yes shure, we can do what´s the best, I have no preferd solution for the naming ;-)